django_redis/cache.py

```diff
@@ -95,7 +98,14 @@ def _get(self, key, default, version, client):
 
     @omit_exception
     def delete(self, *args, **kwargs):
-        return self.client.delete(*args, **kwargs)
+        """returns a boolean instead of int since django version 3.1"""
+        result = self.client.delete(*args, **kwargs)
+        return (
```

Comment:
You can actually compare tuples so `DJANGO_VERSION` > (3, 1, 0) should work

Reply:
done
`        return bool(result) if self._dj_ver >= (3, 1, 0) else result`